app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_catcher.rb

```diff
@@ -1,3 +1,9 @@
 class ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventCatcher < ManageIQ::Providers::BaseManager::EventCatcher
   require_nested :Runner
+
+  def self.all_valid_ems_in_zone
+    super.select do |ems|
+      ems.authentication_key("events").present?
```

Comment:
In a follow-up I think we can also check `ems.authentication_status_ok?(:events)` which would cover the case where someone added an api service key but it isn't valid

Reply:
Sure, will do